sale_order_report_product_image/views/report_saleorder.xml

```diff
@@ -1,27 +1,19 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-
     <template
         id="report_saleorder_document_image"
         inherit_id="sale.report_saleorder_document"
     >
-        <xpath
-            expr="//table[hasclass('o_main_table')]/thead/tr/th[1]"
-            position="before"
-        >
-            <th>Image</th>
+        <xpath expr="//th[@name='th_description']" position="before">
+            <th name="th_image" class="text-left">Image</th>
         </xpath>
-        <xpath
-            expr="//tbody[hasclass('sale_tbody')]//t[@t-as='line']/tr/t[@t-if='not line.display_type']/td[1]"
-            position="before"
-        >
-            <td>
+        <xpath expr="//td[@name='td_name']" position="before">
+            <td name="td_image">
                 <span
-                    t-field="line.product_id.image_small"
+                    t-field="line.product_id.image_128"
```

Comment:
image_small was 64x64, and this one is 128x128, so there will be a difference in the report. Limit the size then to the old one.